### ISSUE TITLE
Playtime graphs no longer work

### ISSUE THREAD

**Issue by Cyprex** [nfreader/slimbus#22]
Personal Role playtime graphs fail to  be generated for me

**Comment by wesoda25** [comment]
Got this too

**Comment by nfreader** [comment]
Investigating

**Comment by nfreader** [comment]
This is an issue with the server's Content Security Policy disallowing inline evaluation of javascript. I'll have to refactor to get around the issue.

**Comment by CoffeeDragon16** [comment]
Yeah, getting this issue as well on Firefox. Only addon is an adblock.

**Comment by ATH1909** [comment]
I got this too.

Also, it looks like whenever I press the Apply Range button, the end date shifts to be one day earlier.
